package.json

```diff
@@ -50,11 +50,14 @@
     "wsrun": "5.2.4"
   },
   "dependencies": {
+    "@headlessui/react": "^1.6.6",
+    "@heroicons/react": "^2.0.10",
     "@types/papaparse": "5.3.2",
     "dayjs": "1.11.0",
     "papaparse": "5.3.1",
     "react-dropzone": "12.0.4",
-    "swr": "1.3.0"
+    "swr": "1.3.0",
+    "tailwindcss": "^3.1.8"
```

Comment:
These changes should be added to the package.json available inside `apps/web`

Reply:
Removed this from package.